docs/devguide/newdashboards.asciidoc

```diff
@@ -30,103 +22,102 @@ The following topics provide more detail about importing and working with Beats
 [[import-dashboards]]
 === Importing Existing Beat Dashboards
 
-You can use the `import_dashboards` script to import all the dashboards and the index pattern for a Beat, including the dependencies such as visualizations and searches.
-The `import_dashboards` script is available under
-https://github.com/elastic/beats/tree/master/dev-tools/cmd/import_dashboards[beats/dev-tools/cmd/import_dashboards]. It's also available in each Beat package under the `scripts` directory.
+You can use the Beat executable to import all the dashboards and the index pattern for a Beat, including the dependencies such as visualizations and searches.
+
+Importing the dashboards can be done as part of the `setup` phase.
+
+
+[source,shell]
+-------------------------
+./metricbeat setup
+-------------------------
+
+The `setup` phase is loading:
+
+- Index mapping template in Elasticsearch
+- Kibana dashboards
+- Machine Learning jobs (if available)
+
+For more details about the `setup` subcommand, run the following command:
+
+[source,shell]
+-------------------------
+./metricbeat help setup   
+                                                                                                                                                             
+This command does initial setup of the environment:
+
+ * Index mapping template in Elasticsearch to ensure fields are mapped.
+ * Kibana dashboards (where available).
+ * ML jobs (where available).
 
-There are a couple of common use cases for importing dashboards:
+Usage:
+  filebeat setup [flags]
 
-* Users who are getting started with Beats may want to <<import-single-beat,import dashboards and/or the index pattern for a single Beat>>.
-* Community Beats developers may want to <<import-dashboards-for-development,import dashboards for development>> to use as a starting point for new dashboards.
+Flags:
+      --dashboards         Setup dashboards only
+  -h, --help               help for setup
+      --machine-learning   Setup machine learning job configurations only
+      --modules string     List of enabled modules (comma separated)
+      --template           Setup index template only
+---------------------------
 
-[[import-single-beat]]
-==== Import Dashboards and/or the Index Pattern for a Single Beat
+The flags are useful in case you don't want to load them all. For example, for importing only the dashboards, you can use the `--dashboards` flag:
 
-Using the `import_dashboards` script from the Beat package, you can import the dashboards and the index pattern to
-Elasticsearch running on localhost for a single Beat (eg. Metricbeat):
+[source,shell]
+---------------------
+./metricbeat setup --dashboards
+-------------------------------
+
+Starting with Beats 6.0.0, the dashboards are not loaded anymore directly into Elasticsearch. They are imported directly to Kibana. 
```

Comment:
Change to say, "Starting with Beats 6.0.0, the dashboards are no longer loaded directly into Elasticsearch. Instead, they are imported directly into Kibana."

Reply:
done